pages/loginPage.js

```diff
@@ -8,7 +10,7 @@ exports.LoginPage = class LoginPage {
   }
 
   async goto() {
-    await this.page.goto('/');
```

Comment:
Why was this change necessary? `'/'` solution is even shown in the Playwright documentation. `BASE_URL` is already defined in the Playwright global config.